src/test/scala/uk/gov/nationalarchives/omega/api/business/agents/ListAgentServiceSummarySpec.scala

```diff
@@ -55,7 +52,7 @@ class ListAgentServiceSummarySpec extends AnyFreeSpec with Matchers {
 
       "an empty payload request" in {
 
-        val listAgentSummaryRequest = ListAgentSummaryRequest()
+        val listAgentSummaryRequest = ListAgentSummary(List())
```

Comment:
For this scenario the stub data is being returned however, after connecting to the database what is the expected results w/ or w/o the depository

Reply:
Good question. I have set this test to be ignored until PACT-1025 is resolved.